aiida/cmdline/commands/cmd_export.py

```diff
@@ -97,6 +97,8 @@ def create(output_file, codes, computers, groups, nodes, input_forward, create_r
     """
     from aiida.orm.importexport import export, export_zip
 
+    echo.echo_critical('the export functionality is currently disabled until issue #2340 is addressed')
```

Comment:
Ehm... won't issue #2340 be closed when merging this PR?
I am confused...

Reply:
By the way: fine to be so explicit about it but I was under the impression that `provenance_redesign` was anyhow not meant to be used for any kind of production runs.

Reply:
You are absolutely right, it should reference #2342
Regarding the disabling of the export. It is true that `provenance_design` is not expected to work, however, we want to merge it soon and it is not clear if #2342 will then have already been addressed. To prevent incorrect archives from being accidentally created, I think it is better to disable it. Still we should fix it as soon as possible of course

Reply:
I have corrected the issue number reference. But I noticed that I forgot to skip tests that either rely on `verdi export create` or on fixture archives that are still `v0.3`. Not ideal, but I think it is still better than people accidentally creating broken archives